fetch.bs

```diff
@@ -732,12 +732,30 @@ a <a for=/>header list</a> <var>list</var>, run these steps:
   <p><a for=list>For each</a> <var>name</var> in <var>names</var>:
 
   <ol>
-   <li><p>Let <var>value</var> be the result of <a for="header list">getting</a> <var>name</var>
-   from <var>list</var>.
+   <li><p>If <var>name</var> is `<code>set-cookie</code>`, then:
+    
+    <ol>
+     <li><p>Let <var>values</var> be a list of all <a lt=value for=header>values</a> of
+      <a for=/>headers</a> in <var>list</var> whose <a for=header>name</a> is a
```

Comment:
https://github.com/whatwg/fetch#formatting

"Do not indent for anything except a new "block" element."